docs/mdx-plugins/remark-link-rewrite.js

```diff
@@ -55,7 +64,8 @@ module.exports = function remarkLinkRewrite(options) {
           newUrl = newUrl.replace(ignoredIndex, '');
         }
 
-        node.url = `/${newUrl}`;
+        // force forward slash on non-posix systems
+        node.url = `/${newUrl.replace(/\\/g, '/')}`;
```

Comment:
On Windows, this would generate a path like: `/path\\to\\file`, which is not correct. There is no internal Node method to force backslashes to forward slashes, so this is our only option. And it does work, just run the tests (before and after this change) on Windows and you'll see these paths are fixed.